time/time.c

```diff
@@ -22,6 +22,8 @@
 #include <stdio.h>
 
 
+#define DAYS_IN_400_YEARS 146097  // The exact number of days in a 400 year period, including all leap day rules.
```

Comment:
Use C style comments
```suggestion
#define DAYS_IN_400_YEARS 146097  /* The exact number of days in a 400 year period, including all leap day rules. */
```